test/yaml_elixir_test.exs

```diff
@@ -214,6 +214,14 @@ defmodule YamlElixirTest do
     )
   end
 
+  test "should receive maps as keyword lists when map is tagged in the file" do
+    assert_parse_file(
+      "keyword_list",
+      %{"prod" => %{"foo" => [{"bar", "foo"}, {"foo", "bar"}]}},
+      node_mods: [YamlElixir.Node.KeywordList]
+    )
+  end
+
```

Comment:
Can you add more test examples to verify different and edge cases?

Reply:
What edge cases are you thinking? Aside from having a yaml doc with a single tagged value, rather than tagging a value in a nested map, what scenarios do you have in mind?

Reply:
In the issue you posted previously, there were more than one case explained. I'm wondering whether there are more examples you can put in the tests.

Reply:
One thing I've done is modify the existing test to have this yaml doc
~~~~
prod:
  foo: !<tag:yaml_elixir,2019:keyword_list>
    foo: bar
    bar: foo
  bar:
    foo: bar
    bar: foo
~~~~

To show that it will turn the tagged value into a keyword list while retaining the untagged value as a map

Reply:
OK